crates/rooch-rpc-server/src/server/rooch_server.rs

```diff
@@ -111,31 +111,28 @@ impl RoochAPIServer for RoochServer {
         access_path: AccessPathView,
```

Comment:
Since the list needs to be removed, consider renaming the fucntion name `list_states` into a unified prefix or suffix, such as `get_states_page` or `query_states`, or is there a better name?
The `list` in other modules are the same as above.

Reply:
I think the function starting with `list` is ok. the result with `List` is repeat with `Page`